core/plugins/org.polarsys.capella.core.data.fa.validation/plugin.xml

```diff
@@ -114,6 +114,25 @@
                                This rule generates a warning if a Function Port and all its connected Functional Exchanges do not have the exact same Exchange Items.
                             </description>
                          </constraint>
+                         <constraint
+                               class="org.polarsys.capella.core.data.fa.validation.functionalExchange.FunctionalExchange_loopsOnSameFunction"
```

Comment:
Can you also update the embedded documentation
https://github.com/eclipse/capella/tree/master/doc/plugins/org.polarsys.capella.validation.doc